### ISSUE TITLE
Document Viewer | Navigating to next document via down arrow

### ISSUE THREAD

**Issue by abbyraskin** [department-of-veterans-affairs/caseflow#2092]
**Story**
When I finish reading a document, I want to use the down arrow to navigate to the next document, so I don't have to switch to another hotkey.

## Acceptance Criteria
- When a user navigates to the last page of a document, they can use the down arrow on their keyboard to open the next document, rather than having to click the right arrow (right arrow also still works).

### Scroll to next document
- When a user scrolls to the next document, it should be in the order starting with the first page of the next document
- When a user scrolls to the next document, a banner slides up with a message that says "You are now entering the next document, [**Insert name of next Doc type in bold**]"   
- The banner stays anchored at the bottom, above the tool bar until the top of the first page of the document is at the top of the document viewer
- When the first page of the document is at the top of the document viewer, the toolbar should change so that doc type matches new document and annotation menu (if open) should also be updated with new doc type information
- Banner specs: message should have 40px margin-left from edge of page, 40px height, color: `primary-alt-lightest` [adapted from styleguide](https://dsva-appeals-certification-dev-1895622301.us-gov-west-1.elb.amazonaws.com/styleguide#colors)

#### Mock up
![image](https://cloud.githubusercontent.com/assets/13420618/26317217/c1003e08-3ee4-11e7-9ae3-608b9bf25f9f.png)

![image](https://cloud.githubusercontent.com/assets/13420618/26317338/1eab1a82-3ee5-11e7-981d-250f021a8c24.png)

### Scroll to previous document
- When a user scrolls to the next document, it should be the last page of the previous document, and continue scrolling in backwards order
- When a user scrolls to the previous document, a banner slides down with a message that says "You are now entering the previous document, [**Insert name of next Doc type in bold**]"   
- The banner stays anchored at the top, above the tool bar until the top of the last page of the previous document is at the top of the document viewer
- When the top of the last page of the previous document hit is at the top of the document viewer, the toolbar should change so that doc type matches new document and annotation menu (if open) should also be updated with new doc type information
- Banner specs: message should have 40px margin-left from edge of page, 40px height, color: `primary-alt-lightest` [adapted from styleguide](https://dsva-appeals-certification-dev-1895622301.us-gov-west-1.elb.amazonaws.com/styleguide#colors)

#### Mock up
![image](https://cloud.githubusercontent.com/assets/13420618/26317292/f9fea46a-3ee4-11e7-8042-01c43fb4880e.png)

![image](https://cloud.githubusercontent.com/assets/13420618/26317317/0d828970-3ee5-11e7-85c9-459cad659bfb.png)

**Comment by abbyraskin** [comment]
cc @gnakm

**Comment by mmlumba** [comment]
@abbyraskin What should the text say (or should there even be a banner) if the user scrolls to the end of the last document in their list, or if the user scrolls to the beginning of the first document in the list?

**Comment by gnakm** [comment]
@mmlumba I don't think we need anything for beginning and end because we have those page indicators that show where users are.

**Comment by abbyraskin** [comment]
@mdbenjam the AC here needs to be updated to reflect our conversation earlier this week

**Comment by abbyraskin** [comment]
@mdbenjam @gnakm @mmlumba 
- We decided to hold on supporting infinite scroll, as we did not feel confident that the less complex experience was worth shipping. 
- We estimate about ~1 week's worth of work to make the code changes necessary to improve infinite scroll to a degree we feel comfortable releasing. 
- After we've released new/improved keyboard shortcuts, we will test with users to evaluate if these improvements sufficiently meet their need to scroll quickly between documents.

Closing this ticket for now.
